cli/src/pcluster/resources/imagebuilder/custom_component_script_s3.yaml

```diff
@@ -0,0 +1,48 @@
+name: ScriptComponent
+description: ScriptComponent
+schemaVersion: 1.0
+
+phases:
+  - name: build
+    steps:
+      # Exit code for failures
+      - name: Fail
+        action: ExecuteBash
+        inputs:
+          commands:
+            - echo 1
+
+      # Script name
+      - name: ScriptName
+        action: ExecuteBash
+        inputs:
+          commands:
+            - |
+              set -v
+              echo {{script-name}}
+
+      # Script url
+      - name: ScriptUrl
+        action: ExecuteBash
+        inputs:
+          commands:
+            - |
+              set -v
+              echo {{script-url}}
+
+      # Download bash script
+      - name: Download
```

Comment:
I'm rethinking about the possibility to have just one component and let it handle the two cases. Let's sync on this

Reply:
done